pom.xml

```diff
@@ -228,6 +228,13 @@
             <artifactId>velocity-tools</artifactId>
             <version>1.4</version>
         </dependency>
+
+        <dependency>
+            <groupId>oro</groupId>
+            <artifactId>oro</artifactId>
+            <version>2.0.8</version>
+        </dependency>
```

Comment:
https://github.com/damianszczepanik/cucumber-reporting#usage here you have some code how to run the report as the standalone app - I use this code and then the code fails.

You can use also this real code from the Jenkins plugin https://github.com/jenkinsci/cucumber-reports-plugin/blob/master/src/main/java/net/masterthought/jenkins/CucumberReportPublisher.java